src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js

```diff
@@ -107,7 +107,9 @@ export default defineComponent({
     newPlaylistDefaultProperties: function () {
       return this.$store.getters.getNewPlaylistDefaultProperties
     },
-
+    locale: function () {
```

Comment:
Is this used?

Reply:
> Adds locale property that was being referenced (but was actually missing) in src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js
https://github.com/FreeTubeApp/FreeTube/blob/8fd5055c4b68fb39a7268c161105f49061f44dd7/src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js#L54-L90

Reply:
Oops, missed that comment.